puzzle/plugins/base_variant_mixin.py

```diff
@@ -0,0 +1,78 @@
+from puzzle.utils import (get_gene_info, get_cytoband_coord)
```

Comment:
I've noticed that you guys surround the imports with `(` and `)`, never seen this before. Any reason? Just curiosity :-)